worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -218,6 +224,7 @@ private MemoryManager(CelebornConf conf) {
 
     if (readBufferThreshold > 0) {
       // if read buffer threshold is zero means that there will be no map data partitions
+      logger.info("start ReadBufferDispatcher:");
```

Comment:
Better not introduce other changes in this pr

Reply:
removed